components/mailgun/sources/new-mailing-list/new-mailing-list.js

```diff
@@ -0,0 +1,67 @@
+const common = require("../common-webhook");
+const moment = require("moment");
+const { mailgun } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-mailing-list",
+  name: "New mailing list",
+  description:
+    "Emit an event when a new mailing list is added to the associated Mailgun account.",
+  version: "0.0.1",
+  dedupe: "greatest",
+  props: {
+    ...common.props,
+    baseRegion: { propDefinition: [mailgun, "baseRegion"] },
+    timer: { propDefinition: [mailgun, "timer"] },
+  },
+  hooks: {
+    async deploy() {
+      // Emits sample events on the first run during deploy.
+      const mailgunLists = await this.mailgun.getMailgunLists("last", 5);
+      if (!mailgunLists || mailgunLists.items.length === 0) {
+        console.log("No data available, skipping iteration");
+        return;
+      }
+      const { items: mailgunListItems = [] } = mailgunLists;
+      mailgunListItems.forEach(this.emitEvent);
+      this.db.set("next", mailgunLists.paging.next);
+    },
+  },
+  methods: {
+    ...common.methods,
+    generateMeta(eventPayload) {
+      const ts = moment(eventPayload.created_at).unix();
```

Comment:
Is `eventPayload.created_at` an ISO 8601 timestamp? You can actually avoid `moment` and use just:

```
+new Date(eventPayload.created_at);
```

Reply:
I checked and it was more of a RFC 2822 Date, but that instruction worked so we don't new moment.